iOS/WeatherToday/WeatherToday/SecondViewController.swift

```diff
@@ -7,66 +7,155 @@
 
 import UIKit
 
-class SecondViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+class SecondViewController: UIViewController, UITableViewDataSource,UITableViewDelegate {
     
     @IBOutlet weak var tableView: UITableView!
     //let cellIdentifier: String = "cell"
     let weatherCellIdentifier: String = "weatherCell"
-    var weatherKr: [WeatherKr] = []
+    // 나라 날씨 배열 데이터
+    var kr: [WeatherKr] = []
+    // 날씨 사진 데이터
+    let weatherImgArr = ["sunny","snowy","rainy","cloudy"]
     
     var textToSet: String?
-    
-    func numberOfSections(in tableView: UITableView) -> Int {
-            return 1 //섹션 두 개 사용
-    }
+
     // 테이블뷰에 섹션에 해당하는 로우의 개수
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return self.weatherKr.count
+        print(self.kr.count)
+        return self.kr.count
     }
     // 해당 로우마다 해당하는 셀을 돌려달라
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         
         let cell: WeatherTableViewCell =
         tableView.dequeueReusableCell(withIdentifier: self.weatherCellIdentifier, for: indexPath) as! WeatherTableViewCell
            
-        let weather: WeatherKr = self.weatherKr[indexPath.row]
+        let weatherKr: WeatherKr = self.kr[indexPath.row]
         
-        cell.cityLabel?.text = weather.city_name
-        cell.stateLabel?.text = "(\(weather.state))" + "/" + "(\(weather.elsius))"
-        cell.rainfallLabel?.text = "(\(weather.rainfall_probability))"
+        cell.cityLabel?.text = weatherKr.city_name
+        
+        // 배열에 있는 값들이 nil일 수도 있음으로 if let 바인딩일 활용한다.
+        var state: Double = 0.0
+        if let stateDouble = weatherKr.state {
+            if let celsiusDouble = weatherKr.celsius {
+                cell.stateLabel?.text = "섭씨 \(stateDouble)도" + " / " + "화씨 \(celsiusDouble)도"
+                state = stateDouble
+            } else {
+                print("error: SecondViewController1")
+            }
+        } else {
+           print("error: SecondViewController2")
+        }
```

Comment:
error의 구분 의도가 명확하지 않은 것 같아서 다음과 같이 변경해도 괜찮을 것 같아요

```suggestion
    if let stateDouble = weatherKr.state, let celsiusDouble = weatherKr.celsius {
        cell.stateLabel?.text = "섭씨 \(stateDouble)도" + " / " + "화씨 \(celsiusDouble)도"
        state = stateDouble
    } else {
       print("error: SecondViewController2")
    }
```